### ISSUE TITLE
network access is disabled

### ISSUE THREAD

**Issue by Shavarsh71** [Stellarium/stellarium#488]
READ THIS AND FILL IN THIS TEMPLATE!

Before reporting an issue around graphics artifacts like missing menu buttons, strange colors, Moon not rendered, or similar, 
*please make absolutely sure* you are running the latest graphics drivers for your graphics card. 
95% of graphics problems are solved this way.

Before reporting an issue, please *absolutely make sure* to check the recent open and also closed issues 
(https://github.com/Stellarium/stellarium/issues?q=is%3Aissue+is%3Aclosed) 
whether it has been reported and solved/closed already! Don't report a new issue in this case, you may join the discussion.

Also look into the wiki https://github.com/Stellarium/stellarium/wiki/Common-Problems-for-the-current-version 
and FAQ https://github.com/Stellarium/stellarium/wiki/FAQ 
and check if you are attempting to report a known issue. Don't report in this case.

Also look into the User Guide before reporting unexpected but correct behaviour. Don't report in this case.

If you miss translations, please help us with your language and join our translators at Transifex: https://www.transifex.com/stellarium/stellarium/dashboard/

If you want to report a feature wish, delete these template lines above and below, and fill in. 

If you want to report a bug, please make sure to update and verify the bug still exists in the current version. 
In this case please also check current beta builds from http://stellarium.org

If the bug still persists, give your issue a clear, short title, delete text above down to and including this line and fill in:

### Actual Behaviour
Describe or maybe attach a screenshot?

### System
* Stellarium version: <Name of downloaded installable file?>
* Operating system: <Name, version number>
* Graphics Card: <Manufacturer (likely Intel, NVidia, AMD?), Model (HD, Geforce, Radeon..., with model number), driver version?>

### Logfile 
If possible, attach the logfile `log.txt` from your user data directory. Look into the Guide for its location.

**Comment by Shavarsh71** [comment]
I want to make a query in on-line catalogue of minor planets and comets. The stellarium shows that network access is disabled while internet is uninterrupted. Checked firewall both and off with result for stellarium.

**Comment by gzotti** [comment]
Please
```
READ THIS AND FILL IN THIS TEMPLATE!
```

above.

**Comment by Shavarsh71** [comment]
Hello,
My stellarium version is 0.18.2. Windows 10. The graphic cards are dual Intel HD Graphics 4400 and Radeon M230.
Irrespective of the firewall off or on position and all permissions granted, the stellarum search dialogue fails to make an on-line query. The same seems to apply to updates of catalogues of minor planets and comets. 
The stellarium claims that network access is disabled. The internet is uninterrupted and fully accessible otherwise to all other programs. 
![stellarium error](https://user-images.githubusercontent.com/43791783/46372181-729e2f00-c69b-11e8-934d-a0fac896a42b.png)

**Comment by gzotti** [comment]
https://github.com/Stellarium/stellarium/wiki/Common-Problems-for-the-current-version describes possible issues around SSH libraries. 
Please try current beta from https://github.com/Stellarium/stellarium-data/releases/tag/beta

**Comment by CF20852** [comment]
I seem to be having a similar problem.  I am running Stellarium beta 0.90.0.16306 under an up-to-date version of Windows 10 Home 64-bit.  My graphics adapters are Intel 4000 and Nvidia GeForce GE 650M (not that it matters for this problem).  I first discovered that I can't update the satellite TLE files using the satellite plugin.  Then I discovered that I can't get star catalog 6 of 9.  The problem appears to be the network interface.  I installed OpenSSL 1.0.2p Light (64-bit), but that doesn't seem to help.  I made sure the Windows firewall is set to allow Stellarium through.  No joy.  My last-run logfile is attached.
[log.txt](https://github.com/Stellarium/stellarium/files/2457043/log.txt).  There are a lot of "Socket operation timed out" errors.

**Comment by Shavarsh71** [comment]
I do not tie this issue with the network interface. Stellarium started to download all the catalogues once I changed it to the latest beta 0.90.
I noticed that the problem developed over the time as initially no such an issue had existed, and stellarium 18.2 had been working all right. The issue kicked off after a certain time span. I could not follow out from which exact moment. 
 No other program depending on internet access has ever been affected by the issue of Stellarium.
